webrender/res/prim_shared.glsl

```diff
@@ -693,4 +693,15 @@ float do_clip() {
     return vClipMaskUvBounds.xy == vClipMaskUvBounds.zw ? 1.0:
         all(inside) ? textureLod(sCache, vClipMaskUv, 0.0).r : 0.0;
 }
+
+vec4 dither(vec4 color) {
+    const float matrix_size = 8;
+    // TODO: should we mask it here or set the texture's wrapping to GL_REPEAT?
```

Comment:
I don't think we need a texture sample at all. We could just `texelFetch` instead